### PR TITLE
Add chown to `/tmp/pacstall` in case it's not owned by user

### PR DESCRIPTION
**Pull request by Henryws**

### CODE REVIEW

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: misc/scripts/install-local.sh [hunk 1]
```diff
@@ -166,6 +166,7 @@ fi
 fancy_message info "Retrieving packages"
 mkdir -p /tmp/pacstall
 cd /tmp/pacstall
+sudo chown -R "$USER":"$USER" /tmp/pacstall
```
[the remark this answers is not in the document]
Comment (Henryws) [reply misc/scripts/install-local.sh:169]: The directories inside dir are covered with the -R part of chown

**Comment by wizard-28** [comment]
Not working
img

**Comment by Henryws** [comment]
Are you trying to run pacstall without sudo?

File: misc/scripts/install-local.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (wizard-28) [reply misc/scripts/install-local.sh]: That -R works for directories present when running the command, but the package directories are created later. So they are not covered by the -R, they will be owned by root most probably.

**Comment by wizard-28** [comment]
> Are you trying to run pacstall without sudo?

No.
this is my build function
img

**Comment by Henryws** [comment]
on the new branch?

**Comment by wizard-28** [comment]
> on the new branch? Also nice colorscheme

Yup this branch, and thanks

**Comment by Henryws** [comment]
which command is breaking it? The cmake . or the other one?

**Comment by wizard-28** [comment]
The other one (the second)

**Comment by Henryws** [comment]
Can't you build without sudo?

**Comment by wizard-28** [comment]
I am building with sudo to build it as the normal user.

**Comment by Henryws** [comment]
> I am building with sudo to build it as the normal user.

Will it not work without sudo?

**Comment by wizard-28** [comment]
It'll install the config files in `/root` instead of `/home/user` otherwise, and this is a widespead issue, currently all pacscripts which install config files (the make command), install it in root (for example neofetch is affected)

**Comment by Henryws** [comment]
So, I need to make pacstall not run as root

**Comment by wizard-28** [comment]
> So, I need to make pacstall not run as root is what your saying?

No, I just want the build function to be able to do its thing as normal user

**Comment by Henryws** [comment]
I'm making a branch that doesn't require pacstall to be started as root (therefore all functions are run as a normal user (besides the sudo ones, but those are temporary))

**Comment by wizard-28** [comment]
I guess thats the way to go as `makepkg` in arch runs as regular user

**Comment by Henryws** [comment]
> > So, I need to make pacstall not run as root is what your saying?
> 
> No, I just want the build function to be able to do its thing as normal user

Uh, the build function runs as a normal user, on line 200, it's just running the pacscripts `build` function

**Comment by wizard-28** [comment]
Still not working

**Comment by Henryws** [comment]
Is this branch even needed anymore?

**Pull request closed without merging**